db/openshift.deploy.yml

```diff
@@ -148,6 +172,9 @@ objects:
             - name: ${NAME}-${ZONE}-${COMPONENT}
               persistentVolumeClaim:
                 claimName: ${NAME}-${ZONE}-${COMPONENT}
+            - name: db-upgrade-backup
+              persistentVolumeClaim:
+                claimName: ${NAME}-${ZONE}-${BACKUP_COMPONENT}-${DB_UPGRADE_VERSION}
```

Comment:
Mount this new db upgrade PVC to OLD database deployment. Later to dump the backup file and be able to restore back to NEW database version.